file-formats/sktd/examples/z_aff_example_sktd.sktd.json

```
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://github.com/SAP/abap-file-formats/file-formats/sktd/sktd.json",
```

`$schema` is obsolete. This shall be replaced by `formatVersion`:

```suggestion
  "formatVersion": "1",
```